understand_data.R

```diff
@@ -0,0 +1,19 @@
+# Glimpse of data and missing value check
+understand_data<- function(data) {
+    glimpse(data)
+    
+    check_missing = levels(factor(is.na(data)))
+    if(length(check_missing) <= 1 && check_missing != TRUE) {
+        cat(blue("There are no missing values in your dataset."))
+        } else {
```

Comment:
nit: here and elsewhere use consistent indentation

You can automate finding/fixing the nit-picky stuff using the R packages described here: https://style.tidyverse.org/